packages/cli/src/services/communityPackages.service.ts

```diff
@@ -325,13 +324,22 @@ export class CommunityPackagesService {
 		await this.orchestrationService.publish('community-package-uninstall', { packageName });
 	}
 
+	private getNpmRegistry() {
+		let { registry } = this.globalConfig.nodes.communityPackages;
+		if (registry !== DEFAULT_REGISTRY && !this.license.isCustomNpmRegistryEnabled()) {
+			this.logger.warn('Not licensed to use custom NPM registry');
+			registry = DEFAULT_REGISTRY;
```

Comment:
A paying user whose license expires might inadvertently end up installing or updating from `DEFAULT_REGISTRY`.

Should we throw `FeatureNotLicensedError` instead and ask them to remove `N8N_COMMUNITY_PACKAGES_REGISTRY` or renew the license?